src/omp/backend.f90

```diff
@@ -233,30 +233,29 @@ subroutine transeq_halo_exchange(self, u, v, w, dir)
 
   end subroutine transeq_halo_exchange
 
-  subroutine transeq_dist_component(self, rhs, u, conv, &
+  subroutine transeq_dist_component(self, rhs_du, u, conv, &
                                     u_recv_s, u_recv_e, &
                                     conv_recv_s, conv_recv_e, &
                                     tdsops_du, tdsops_dud, tdsops_d2u, dir)
       !! Computes RHS_x^u following:
       !!
       !! rhs_x^u = -0.5*(conv*du/dx + d(u*conv)/dx) + nu*d2u/dx2
```

Comment:
And likewise, same the changed `intent` implies the equation described in the comment needs to be updated.